settings.gradle

```diff
@@ -30,3 +30,10 @@ include ':feature-nft-api'
 include ':feature-nft-impl'
 include ':feature-liquiditypools-api'
 include ':feature-liquiditypools-impl'
+
+// Map fearless-utils-Android from GitHub as a source dependency (requires network access)
```

Comment:
Maybe it would be useful to clarify why we handle the fearless-utils-Android dependency in a special way.